docs/source/usage/workflows/boundaryConditions.rst

```diff
@@ -15,15 +15,20 @@ Particles
 
 By default, boundary kinds match the value of ``--periodic``.
 For species with a particle pusher, it can be overridden with option `<prefix>_boundary <x> <y> <z>`.
-However, currently we do not support options that do not match the default value.
```

Comment:
Could you add that absorbing and reflecting particle boundaries are only allowed when the same boundary for fields is absorbing